### ISSUE TITLE
Pointer drag events are offset by output position

### ISSUE THREAD

**Issue by edrex** [ifreund/river#186]
Pointer drag events (not clicks) are offset on my translated output. 

My multi-display setup with a hidpi laptop screen to the right of an external screen rotated portrait is configured like:

```
> wlr-randr --output eDP-1 --pos 1080,240 --scale 2 --output DP-1 --transform 90
> wlr-randr
eDP-1
  Mode: 3200x1800 px (current)
  Position: 1080,240
  Transform: normal
  Scale: 2.000000
DP-1
  Mode: 1920x1080 px (current)
  Position: 0,0
  Transform: 90
  Scale: 1.000000
```
On the translated screen, events are translated to the right and down. I am doing rotation, scale, and translation together with a multi-display setup, so I'm not sure which combination of these is triggering the issue. I will experiment with different configs.

@maringuu #143

**Comment by edrex** [comment]
Some more data:

 * **DP-1** `--transform normal` the behavior on `eDP-1` is the same (so not rotation related)
 * with **DP-1** right of **eDP-1** drag events are positioned correctly on both displays, so it seems this only happens with a HiDPI display that is positioned away from the origin
 * With `eDP-1 --scale 1` the drag events are still offset, but by half as much as with `--scale 2`, so the effect is scale dependent.

**Comment by edrex** [comment]
I played with this a bit more and realized that this happens whenever the output is offset from the origin, and the drag events are just missing the offset (so if you drag up and left to start from the origin of the coordinate system, you will be at the top left of the screen you started from.

Also worth noting, the cursor is in the correct position while dragging.

**Comment by ifreund** [comment]
Thanks for the detailed bug report/investigation, this should be fixed by b468f0aa8502e0cec985f4a9f799c292565f5dcf

**Comment by edrex** [comment]
Confirmed fixed with that commit. 

Thanks for river and for your contributions to wlroots!
